src/webextension/list/manage/components/item-details.css

```diff
@@ -3,7 +3,10 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 .item-details {
-  margin: 2em;
+  padding: 2em;
+  background-color: #f3f3f5;
+  height: 100%;
```

Comment:
>The spec shows that all the empty space below the form should be the same color, but in practice, the background-color defined above only applies to the form area.

You're right. And I struggled to see why/how to fix it with a quick glance. I assume it has to do with grid/flex? If you have any pointers I'd appreciate it, otherwise I'll keep hunting around.

Reply:
>And I struggled to see why/how to fix it with a quick glance. I assume it has to do with grid/flex?

Wait, never mind, I think I see it now. Removing the "height" and set the right background-color.